From 5367c10ccf59c25a48d868138bd4e511e70b0dcf Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Thu, 15 Feb 2018 14:31:17 +0000
Subject: [PATCH] OS-6591 VMX support is not checking for PAT load/save support
 Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/uts/i86pc/io/vmm/intel/vmx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/uts/i86pc/io/vmm/intel/vmx.c b/usr/src/uts/i86pc/io/vmm/intel/vmx.c
index 3aa1d473d6..2689766193 100644
--- a/usr/src/uts/i86pc/io/vmm/intel/vmx.c
+++ b/usr/src/uts/i86pc/io/vmm/intel/vmx.c
@@ -130,6 +130,7 @@ __FBSDID("$FreeBSD$");
 
 #define	VM_EXIT_CTLS_ONE_SETTING					\
 	(VM_EXIT_HOST_LMA			|			\
+	VM_EXIT_LOAD_PAT			|			\
 	VM_EXIT_SAVE_EFER			|			\
 	VM_EXIT_LOAD_EFER			|			\
 	VM_EXIT_ACKNOWLEDGE_INTERRUPT)
-- 
2.21.0

